test/check-application

```diff
@@ -2081,6 +2083,21 @@ class TestApplication(testlib.MachineCase):
             podmanRestartEnabled = self.execute(False, "systemctl --user is-enabled podman-restart.service || true").strip()
         self.assertEqual(podmanRestartEnabled, 'enabled')
 
+    def testLingerPolicyChanges(self):
+        """Test that watching for a lingering file in /var/lib/systemd/linger keeps working"""
+
+        b = self.browser
+        self.login(False)
+
+        b.click("#containers-images button.pf-c-expandable-section__toggle")
+        self.machine.execute("loginctl enable-linger $(id -u admin)")
+        self.addCleanup(self.machine.execute, "loginctl disable-linger $(id -u admin)")
+
+        b.wait_visible('#containers-images td[data-label="Image"]:contains("busybox:latest")')
+        b.click('#containers-images tbody tr:contains("busybox:latest") .ct-container-create')
+        b.wait_visible('div.pf-c-modal-box header:contains("Create container")')
+        b.wait_visible("#run-image-dialog-restart-policy")
```

Comment:
Hmm, this is visible even if podman-restart.service isn't available?

Reply:
Good point, we do check for it for the system service, but not the user service yet. Also a good point is to check when we can drop that..